clean_air/util/file_converter.py

```diff
@@ -0,0 +1,106 @@
+"""This converts xcel data files to JSON file types for processing."""
+import pandas as pd
+import openpyxl
```

Comment:
I don't think this import is needed? Nothing from the module is used, there's just a keyword argument `engine='openpyxl'` requesting pandas to use it.

If the environment didn't have `openpyxl` at all, add it to `environment.yaml`.

Reply:
I needed to import openpyxl when writing this otherwise it didn't work.  I'll try removing it in case I no longer need it, but I definitely did to begin with.

I will also add both pandas and openpyxl to environment.yaml (had forgotten how to do that).

Reply:
I think that's pycharm being clever and adding packages to the environment for you, as well as import statements, if it sees that you're using something that isn't there. Pandas was definitely already in the enivronment though - are you up to date?

Reply:
Maybe not, as I had to add it to my own cap_env using conda.  I did think that I'd synced the codebase properly though, where does environment.yaml live?

Reply:
environment.yaml is just at the root of the repository

Reply:
It was fine removing openpyxl from the imports and I didn't actually have to add anything to the environment.yaml (which was very easy to find, btw).

Reply:
No you still do need to add it to the environment.yaml. Pycharm added `openpyxl` to your environment, but has no idea it was supposed to add it to that file too. Likewise, adding anything to environment.yaml would not affect the current environment (you'd run `conda env update --file environment.yaml`), but it would mean that anyone making a new environment in future would not get everything they need.

Reply:
Right, I've missed the constant confusion and complexity of using conda.  I had manually added openpyxl and pandas to my pycharm working environment, and knew at that time that it was going to be confusing later.  I must be using an environment which has the same name as our main one (cap_env?) and is largely the same, but doesn't keep up to date with the environment.yaml (because I have to update it myself and I haven't, presumably...).

Anyway, regardless of all that I will just go ahead and add openpyxl to the environent.yaml.  Thank you for your help with this.